src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -71,6 +71,33 @@ AttributRadiationFlag
 
 /*########################### define species #################################*/
 
+/*--------------------------- photons -------------------------------------------*/
+
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
+
+value_identifier(float_X, DummyMass, 1.0);
+value_identifier(float_X, DummyCharge, 0.0);
+
+typedef bmpl::vector<
+    particlePusher<particles::pusher::Photon>,
+    shape<UsedParticleShape>,
+    interpolation<UsedField2Particle>,
```

Comment:
Your pusher does not need a interpolation. Would this be necessary later for interactions with the density field?